### ISSUE TITLE
pages hooks do not work outside of monorepo

### ISSUE THREAD

**Issue by gautamsi** [keystonejs/keystone#1985]
# Bug report

## Describe the bug
Hooks #1927  for pages do not work outside of monorepo.

## To Reproduce

 copy the blog demo to a different folder outside of monorepo, install dependencies and run `yarn dev`. Notice missing sidebar links to custom pages.

## Expected behaviour

Should see custom pages links in the sidebar.

## Screenshots

notice missing links to custom about and dashboard page.
![image](https://user-images.githubusercontent.com/5769869/69242013-009de780-0bc6-11ea-822a-4d5a3ec08e1b.png)

## System information

- OS: Windows (not tried on Mac outside the monorepo)
- Browser: Chrome

**Comment by LiamAttClarke** [comment]
Experienced the same issue working on MacOS (browser: Chrome).

**Comment by LiamAttClarke** [comment]
Reverting to the old API fixed this for me:
```js
new AdminUIApp({
  pages: [
    {
      label: 'Custom Dashboard',
      path: '',
      component: require.resolve('./admin/pages/CustomDashboard')
    }
  ]
})
```

**Comment by MadeByMike** [comment]
I found the cause for this and it should be resolved in next release

**Comment by gautamsi** [comment]
Is this closed due to wrong reference in #2003?
